logdna/client.go

```diff
@@ -14,7 +14,8 @@ type Client struct {
 	HTTPClient *http.Client
 }
 
-type viewPayload struct {
+// ViewPayload contains view-specific data- including apps and categories and is forwarded to the config-api
+type ViewPayload struct {
```

Comment:
why is this being capitalized and exported while the others are not? IMO they should all be exported, or none should be. it doesn't look like they need to be though.

Reply:
ViewPayload needs to be accessed from resource_alert and resource_client.go whereas the others only need to be accessed within client.go. I'll capitalize everything for consistency

https://github.com/logdna/terraform-provider-logdna/pull/3/files#diff-0b4c49f154f7dc1f04a503de890c274c777876182f3bd5e499113009e172e19fR24

Reply:
these are all in the same package `logdna` though